myem_lib/utils.py

```diff
@@ -1,126 +1,70 @@
 """Utils."""
-import json
 import os
 from typing import Any, Dict
 
 import jwt
-from jwt import PyJWKClient
-from nameko.web.handlers import HttpRequestHandler
-from werkzeug import Response
-from myem_lib.exceptions import BadRequest, HttpError, Unauthenticated, Unauthorized
-from functools import partial
-from nameko.extensions import register_entrypoint
+from fastapi import Depends, FastAPI, HTTPException, Request
+from fastapi.exceptions import RequestValidationError
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.responses import JSONResponse
+from fastapi.security import OAuth2PasswordBearer
 
 
-class HttpEntrypoint(HttpRequestHandler):
-    """Control formatting of errors returned from the service by overriding response_from_exception.
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="token")
```

Comment:
we should use fastapi users dependencie insead.
https://fastapi-users.github.io/fastapi-users/usage/current-user/
It inject all needed things directly. do we really need to wrap it ?

Reply:
we need to ckeck how to handle this in micro service way.

Reply:
lets discuss about it.